jekyll/_cci2/gh-bb-integration.md

```diff
@@ -224,15 +184,31 @@ If you enable these restrictions on an organization for which CircleCI has been
 
 The account and permissions system we use is not as clear as we would like and as mentioned we have a much improved system in development with users as first class citizens in CircleCI.
 
+## Bitbucket API and Organization Changes
+
+A number of Bitbucket API endpoints and response payloads have recently been deprecated; including the `/2.0/teams` and most `/2.0/users` endpoints. If you are using Bitbucket, you will need to migrate to the new `/2.0/workspaces` endpoints.
+
+This change was made to enable org administrators, using multiple Atlassian cloud products, to enforce access to their content across their many different product entities. With this transition, you will see a limited set of expansions to read permissions for users who have only repo access (i.e. are not members of a group) within a given Bitbucket workspace. Those users will now have read access to the following:
+
+* Pages
+  * Org Settings
+  * Plans
+
+* API endpoints
+  * `GET /v1/organization/{name}/settings`
+  * `GET /v1/organization/{name}/plan`
+  * Workspace included in `GET /v1/user/organizations` response
+
+**Note:** "Workspace"s are mapped to organizations in CircleCI in the same manner that "teams" were previously. Bitbucket now considers users with only repo access to be “members” of the workspace per their new API endpoints, which has led to the permission change. 
+
+No changes on your end should be necessary for existing teams/workspaces to continue to function as organizations within CircleCI.
+
```

Comment:
@thomasr8 it looks like this section is the only non-whitespace change in the PR - is that right?

Reply:
+1 to what @mzalimeni said. Ideally, we just update docs to match new behaviour and do not include the announcement.